packages/treasury/src/stablecoinMachine.js

```diff
@@ -118,11 +151,15 @@ export async function start(zcf, privateArgs) {
     { Central: runIssuer },
     {
       timer: timerService,
-      poolFee: loanParams.poolFee,
-      protocolFee: loanParams.protocolFee,
+      // TODO(hibbert): make the AMM use a paramManager. For now, the values
+      //  are fixed after creation of an autoswap instance.
+      poolFee: feeParams.getParam(POOL_FEE_KEY).value,
+      protocolFee: feeParams.getParam(PROTOCOL_FEE_KEY).value,
     },
   );
 
+  const poolParamManagers = makeStore('brand'); // Brand -> poolGovernor
```

Comment:
might as well formalize the type declaration, I think:

```suggestion
  /** @type { Store<Brand, PoolParamManager> } */
  const poolParamManagers = makeStore('brand');
```

Reply:
done